Lavinia-api/Data/ElectionContext.cs

```diff
@@ -13,10 +13,17 @@ public ElectionContext(DbContextOptions<ElectionContext> options) : base(options
         {
         }
 
+        // API v1
```

Comment:
It would have made more sense to create a new Context, perhaps NorwayContext as all the data inserted here is Norway specific.